Makefile

```diff
@@ -58,6 +58,10 @@ RESET_REGRESSION?=false
 tests: bin binaries tests/tests.sh
 	$(Q)./tests/tests.sh $(RESET_REGRESSION)
 
+.PHONY: tezt
+tezt: bin binaries
```

Comment:
Do we need two targets? Don't we need only one test target to launch all tests (to limit entry points that are visible by users)?

Reply:
For the CI, I think it is best for the moment. When all the tests will be integrated with `Tezt`, the new target won't be needed anymore.